docs/bashwarning.md

```diff
@@ -0,0 +1,15 @@
+# BashV3 Warning
+
+As of February 2018, the BashV3 task has started executing scripts with a target script type of "File Path" instead of sourcing them in the default case.
+This does not work for scripts that do not have the executable bit set.
+Due the potential for breaking existing task users with this change, we have decided to continue to source scripts that don't have the executable bit set but throw a warning.
```

Comment:
"For backwards compatibility, we will continue to source scripts that don't have the executable bit set. This will throw a warning."

Reply:
That's much better, thanks.